plugins/CustomDimensions/tests/System/ApiTest.php

```diff
@@ -264,6 +264,10 @@ public function getApiForTesting()
 
     public function testScheduledReport()
     {
+        //todo this should be removed once github action is alive.
+        if (!getenv('GITHUB')) {
+            $this->markTestSkipped("Github and Travis has some fonts diff, please see Github Action tests");
+        }
```

Comment:
Might be better to check with `Fixture::canImagesBeIncludedInScheduledReports()` if the test should be run or not.
I guess the reason for the differing result might be a different gd lib version. The method above should check that accordingly.